js/code-snippets.js

```diff
@@ -4,21 +4,22 @@ const codeSnippets = {
 `const dbUri = process.env.MONGODB_URI || 'mydb';
 const collections = ['mycollection'];
 
+// eslint-disable-next-line
 const db = mongojs(dbUri, collections);`,
 
   sequelizeCode:
 `// Run sequelize before listen
-db.sequelize.sync({ force: true }).then(function() {
-    app.listen(port, function() {
-    console.log("App listening on PORT " + port);
-    });
+db.sequelize.sync({ force: true }).then(() => {
+  app.listen(port, () => {
+    console.log(\`App listening on PORT \${port}\`);
```

Comment:
Just curious. what is this `\` doing in the beginning of console.log()

Reply:
the linter expects strings to be templates not concatenated
so since this is already a template literal, i need to escape the inner one so that it prints out a back tick in the generated file